source/reference/sort-bar.html

```diff
@@ -0,0 +1,80 @@
+<div>
+  <label for="field">
```

Comment:
Not too sure if this is the right place to put it but can we fix the functionality on the sort end? To me it is unintuitive for the toggle sort and toggle reverse to be different things, as its confusing when you click the arrow only and the arrow moves but the scroll isnt toggled, I think a possible better interaction would be a three click toggle, where clicks change state (ex click one to sort normally, click two to sort reverse, click three to revert to not sorting state)

Reply:
Yeah, good idea. Made the changes and it works as described:
![image](https://user-images.githubusercontent.com/97627312/204592184-fc24dd5c-1faf-4ca1-80c3-baafefe9f321.png)
![image](https://user-images.githubusercontent.com/97627312/204592220-5042cdc6-a950-4edc-ae2e-eb808af6afc7.png)
![image](https://user-images.githubusercontent.com/97627312/204592248-bd5b3087-5343-4c73-8f4a-ffe1962d483e.png)
![image](https://user-images.githubusercontent.com/97627312/204592282-f63ff044-92bd-47d1-a118-2c5fd28cd6a0.png)